arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/ar/ArcGISArView.kt

```diff
@@ -655,6 +661,31 @@ class ArcGISArView : FrameLayout, DefaultLifecycleObserver, Scene.OnUpdateListen
         return false
     }
 
+    /**
+     * Performs a hitTest at the [screenPoint]. If the hitTest succeeds this returns the point in the
+     * world where the hitTest happened
```

Comment:
instead of "world" better use the term "scene"?

Reply:
Missing full stop .

Reply:
I've changed it to 'AR scene' to differentiate between the ArcGIS scene and AR scene.